templates/application_request/request.html.twig

```diff
@@ -103,6 +100,11 @@
     <div class="information__modal inf-modl--pale-blue">
         Ce formulaire n’engage pas une présence sur une liste municipale ou une implication automatique dans une campagne locale.
     </div>
+    <br>
```

Comment:
@enzozccr , reminder:
maybe you will want to remove this `<br>` and do CSS tricks here